pkg/utils/kubernetes/crd.go

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
```

Not sure if both is possible, though. (cc @rfranzke)

Reply:
In the repository, I see both represented. I used the Makefile target ` add-license-headers` to generate it.
Which one should I use?

Reply:
If `make add-license-headers` added it this way, I'd say it's fine for now.